source/common/vue/form/elements/Checkbox.vue

```diff
@@ -1,16 +1,26 @@
 <template>
-  <div class="form-control cb-group">
-    <label class="checkbox">
-      <input
-        v-bind:id="fieldID"
-        type="checkbox" v-bind:name="name" value="yes"
-        v-bind:checked="value"
-        v-bind:disabled="disabled"
-        v-on:input="$emit('input', $event.target.checked)"
+  <div>
+    <div class="form-control cb-group">
+      <label class="checkbox"
+             v-bind:disabled="disabled"
       >
-      <span class="checkmark"></span>
-    </label>
-    <label v-if="label" v-bind:for="fieldID" v-html="label"></label>
+        <input
+          v-bind:id="fieldID"
+          type="checkbox" v-bind:name="name" value="yes"
+          v-bind:checked="value"
+          v-bind:disabled="disabled"
+          v-on:input="$emit('input', $event.target.checked)"
+        >
+        <span class="checkmark"></span>
+      </label>
+      <label v-if="label" v-bind:for="fieldID" v-bind:disabled="disabled"
+             v-html="label"
+      >
+      </label>
+    </div>
+    <div v-if="info" class="form-control">
+      {{ info }}
```

Comment:
You're using this as an info to display the error, right? I have to take a look at this how it looks …

Reply:
This is a screenshot on Windows. Do you like the look?
![Preferences with Gnome error](https://user-images.githubusercontent.com/5193990/117012082-b4992100-ad2d-11eb-8ee2-f03eb8923839.png)

Reply:
Interesting that this icon works like an Emoji :O I did not expect that!

But yeah, in general this looks good – albeit I would make the text much smaller and also grey – now it looks as if it didn't belong to the checkbox. I've had something like that in previous versions which I not yet restored … wait let me search for a screenshot.

Here: https://docs.zettlr.com/en/reference/settings/#advanced

The Advanced screenshot features two inputs with such info strings – they are very close to the input they refer to and a little bit grey. This way this warning wouldn't draw so much attention (it's only important for those who want the tray anyway)